src/python/WMCore/WMRuntime/Watchdog.py

```diff
@@ -103,8 +104,8 @@ def setupMonitors(self, task, wmbsJob):
                 # If we did base maxRSS off the memory in the HTCondor slot, subtract a bit
                 # off the top so watchdog triggers before HTCondor does.
                 # Add the new number of cores to the args such that DashboardInterface can see it
+                args['cores'] = resources['cores']
```

Comment:
Actually I was wondering whether we should be put below instead.

args['cores'] = changedCores

Not sure which core number we should propagate.

Reply:
changedCores is just a boolean value. Before calling `resizeResources`, `resources` has the original number of cores, once it comes back, it might have a different value (resources is passed by reference).